acceptance/scripts/hosts.cfg

```diff
@@ -1,37 +0,0 @@
----
```

Comment:
Yeah this was only ever meant to be a transient `hosts.cfg` anyway, looks like an errant `git add` included it. I recommend adding it to `.gitignore`.